repo.sh

```diff
@@ -234,7 +234,7 @@ status ()
             printf "\nGit status for [%s]:\n" "$name"
             cd "$name";git status;cd "$currDir"
         else
-            printf "The [%s] repo is not cloned. Skipping.\n" "$name"
+            echo "Skipped repo since name ($name) did not match an existing directory: $repo"
```

Comment:
How would this typically get resolved? Running `make dev.clone`, or other? The earlier message hinted at a next step. Maybe we make it explicit? Or am I entirely off about what issue this represents?

Reply:
Ah, good point. It would either need to get cloned or the URL would need to get fixed. I'll go for the additional check after all (which will allow a more explicit message here).

Reply:
Reworked to handle this differently. I'm not sure why, but in some cases failure to match the regex would result in an immediate exit, and other times the match would just not be set. (Depended on whether I had the end-anchor character in there, I think?) So I just handled failure explicitly, which is probably better anyhow.